src/test/java/com/flightstats/hub/cluster/LastContentPathTest.java

```diff
@@ -26,8 +28,8 @@ public static void setUpClass() throws Exception {
     }
 
     @Before
-    public void setUp() throws Exception {
-        lastContentPath = new LastContentPath(curator);
+    public void setUp() {
+        lastContentPath = new LastContentPath(curator, new AppProperty(PropertyLoader.getInstance()));
```

Comment:
Would it be possible to mock AppProperty here instead? ...totally understandable (though sad) if this is actually an integration test, but if not...